src/spoof.ts

```diff
@@ -766,10 +746,111 @@ export const createCursor = (
           behavior: scrollSpeed < 90 ? 'smooth' : undefined
         }))
       }
+    },
+
+    async scroll (delta: Partial<Vector>, options?: ScrollOptions) {
```

Comment:
Basically just moved this code (unchanged) to its own function that is called by both `scrollTo` and `scrollIntoView`